docs/index.md

```diff
@@ -1,4 +1,51 @@
 # Ansible PROJECTNAME Documentation
 
-Provide a brief summary of the Ansible project and why someone would want to use it.
-The summary should be two or three sentences long.
\ No newline at end of file
+> Provide a mission statement or a brief summary of the Ansible project and why someone would want to use it. It should be two or three sentences long, for example:
+
+At the `PROJECTNAME`, our mission is to produce and maintain simple, flexible,
+and powerful open-source software tailored to `PROJECT_PURPOSE`.
+
+We welcome members from all skill levels to participate actively in our open, inclusive, and vibrant community.
+Whether you are an expert or just beginning your journey with Ansible and `PROJECT_NAME`,
+you are encouraged to contribute, share insights, and collaborate with fellow enthusiasts!
+
+> Please have at least the following sections filled up with relevant information. Depending of your project complexity, you can add additional sections, for example, a separate Getting started guide and so on.
+
+- [License](LICENSE link here)
+- [Community guide](community_guide.md)
+  - [Code of Conduct](community_guide.md#code-of-conduct)
+  - [Communication](community_guide.md#communication)
+    - [Asynchronous channels](community_guide.md#asynchronous-channels)
+    - [Real-time channels](community_guide.md#real-time-channels)
+
+- [User guide](user_guide.md)
+  - [Communication](user_guide.md#communication)
```

Comment:
I'm not sure we need this repeated here. Communications should stay in the community guide.

Reply:
Yep, it looks ugly, removed, thanks